src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -131,9 +147,77 @@ AndroidDeviceControllerWrapper * AndroidDeviceControllerWrapper::AllocateNew(
     chip::Controller::AndroidOperationalCredentialsIssuer * opCredsIssuer = wrapper->mOpCredsIssuer.get();
 
     // Initialize device attestation verifier
-    // TODO: Replace testingRootStore with a AttestationTrustStore that has the necessary official PAA roots available
-    const chip::Credentials::AttestationTrustStore * testingRootStore = chip::Credentials::GetTestAttestationTrustStore();
-    SetDeviceAttestationVerifier(GetDefaultDACVerifier(testingRootStore));
+    const Credentials::AttestationTrustStore * trustStore;
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    if (paaCertsArrayList)
+    {
+        jint listSize;
+        JniReferences::GetInstance().GetListSize(paaCertsArrayList, listSize);
+        std::vector<std::vector<uint8_t>> paaCerts;
+        for (uint8_t i = 0; i < listSize; i++)
```

Comment:
Please use size_t or jint instead of uint8_t for the indexing. Otherwise, you will run out of entries if there are > 255